### PR TITLE
Added myself to contributors

### PR DESCRIPTION
**Pull request by KeinsWolfi**
Added myself because im contributor now (added PrimalFear solvers in PR #2771)

### CODE REVIEW

**Comment by KeinsWolfi** [comment]
oh and my discord is @einswolfi2 if you need that

File: constants/Contributors.json [hunk 1]
```diff
@@ -67,6 +67,11 @@
       "suffix": "§d:3"
     },
     "oxsss": {},
+    "PlasticEating": {
+      "suffix": "§d§l:3",
+      "spinny": true,
```
Comment (hannibal002) [inline constants/Contributors.json:72]: please dont use spinny and upside down

**Review by hannibal002 — CHANGES_REQUESTED** [review]

File: constants/Contributors.json (hunk 1 above)
Comment (KeinsWolfi) [reply constants/Contributors.json:72]: No Problem

Comment (hannibal002) [reply constants/Contributors.json]: same with upside down

Comment (KeinsWolfi) [reply constants/Contributors.json]: Ahhh I just thought cuz a few others had it still

**Pull request merged**